File: day5/go/main.go
package main

import "fmt"

func test_action() {
	fmt.Println("Hello World")
}

func test_action1() int {
	return 1
}

func test_action2(value int) int {
	return value
}

func main() {
	test_action()
	var value int = test_action1()
	fmt.Println(value)
	var value1 int = test_action2(1)
	fmt.Println(value1)
}

File: day5/rust/main.rs
fn test_action() {
	println!("test")
}

fn test_action1(value:String) {
	println!("{}", value)
}

fn test_action2(value:i32) -> i32 {
	value
}

fn main() {
	test_action();
	test_action1(String::from("test"));
	println!("{}", test_action2(1));
}

